go.mod

```diff
@@ -1,11 +1,17 @@
 module github.com/ssut/payload-dumper-go
 
-go 1.14
+go 1.18
```

Comment:
Just a simple question: why did you choose Go 1.18 instead of the latest stable release(Go 1.22)? Is there any special reason for this?

Reply:
Usually the `go` directive is about the minimal required version to run the software.

In the case of libraries, you always want it to be compatible with as many clients as possible, so it's usually set to the lowest possible value according to the language features you require in the code.

But, since it's an application here, the owner may set it to whichever version they like. I've just updated it to the minimal required version (since one of the dependencies now uses generics introduced in 1.18).

Reply:
So yeah, if you agree, I can update it to 1.22 :)

Reply:
Thanks for your explanation in detail. Well I think the current state is okay.